app/controllers/legal_docs_controller.rb

```diff
@@ -0,0 +1,44 @@
+class LegalDocsController < ApplicationController
+  before_filter :find_legal_doc, except: [:index, :new, :create]
```

Comment:
That's definitely the typical flow, for sure. 

Our use-case here is just a little different. On the backend (as admins), we will want to see a list. But when someone clicks 'privacy' -- we just want to take them to the latest Privacy policy page.